frontend/src/pages/ShootDetails.vue

```diff
@@ -146,20 +146,24 @@ export default {
   methods: {
     ...mapActions([
       'getShootAddonKyma'
-    ])
+    ]),
+    fetchKymaInfo () {
+      if (!this.isKymaFeatureEnabled && this.isKymaAddonEnabled) {
+        return
+      }
+      this.getShootAddonKyma({ name: this.shootName, namespace: this.shootNamespace })
+    }
   },
   mounted () {
     if (get(this.$route, 'name') === 'ShootItemHibernationSettings') {
       this.$refs.shootLifecycle.showHibernationConfigurationDialog()
     }
+    this.fetchKymaInfo()
   },
   watch: {
     isKymaAddonEnabled (currentlyEnabled, previouslyEnabled) {
-      if (!this.isKymaFeatureEnabled) {
-        return
-      }
       if (!previouslyEnabled && currentlyEnabled) {
```

Comment:
why check the previous value? If current value is true and it has changed we should fetch the infi.

Reply:
✅